models/service_detail.go

```diff
@@ -0,0 +1,246 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+//
+// Copyright NetFoundry, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// __          __              _
+// \ \        / /             (_)
+//  \ \  /\  / /_ _ _ __ _ __  _ _ __   __ _
+//   \ \/  \/ / _` | '__| '_ \| | '_ \ / _` |
+//    \  /\  / (_| | |  | | | | | | | | (_| | : This file is generated, do not edit it.
+//     \/  \/ \__,_|_|  |_| |_|_|_| |_|\__, |
+//                                      __/ |
+//                                     |___/
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"strconv"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/go-openapi/validate"
+)
+
```

Comment:
Service Detail doesn't have the `config` attribute with the actual embedded configuration data for the config types the user expressed interest in. Does that need to be added or are you planning a different API to get that data?

Reply:
Switched back to array of strings.

Reply:
Ah, so you saw a different incompatibility than I did. I think change from []string -> []*EntityRef for configs is fine and an improvement, even it's not backwards compatible. I think we should probably actually remove configs all together, since you can get them using /services/<id>/configs. 

What I was concerned about was config, which embeds the actual config data related to a service in the service definition. So you'll see something like this:
```
            "name": "iperf",
            "terminatorStrategy": "",
            "roleAttributes": null,
            "permissions": [
                "Bind",
                "Dial"
            ],
            "configs": [
                "1a6290b4-39fb-4bf0-9eac-3e3fb2906029"
            ],
            "config": {
                "ziti-tunneler-client.v1": {
                    "hostname": "iperf",
                    "port": 5201
                }
            }
```
That still needs to be there, b/c non-admins need to be able to get the config related to the service.